src/vim9class.c

```diff
@@ -245,8 +247,28 @@ object_index_from_itf_index(class_T *itf, int is_method, int idx, class_T *cl)
 					      cl->class_name, itf->class_name);
 	return 0;
     }
-    int *table = (int *)(i2c + 1);
-    return table[idx];
+    if (is_static)
+    {
+	// TODO: Need a table for fast lookup?
+	char_u *name = itf->class_class_members[idx].ocm_name;
+	for (int i = 0; i < i2c->i2c_class->class_class_member_count; ++i)
+	{
+	    ocmember_T *m = &i2c->i2c_class->class_class_members[i];
+	    if (STRCMP(name, m->ocm_name) == 0)
```

Comment:
Looking at this code now, comparing the name of every class member variable is expensive to do at run time for every access.  Is it possible to move this to compile time and use a lookup table?

Reply:
> comparing the name of every class member variable is expensive to do at run time

I agree, that's why I left the comment
```
    if (is_static)
    {
	// TODO: Need a table for fast lookup?
```

>  Is it possible to move this to compile time and use a lookup table?

Probably, but I don't know how the table used for non-static is built. First thing was to get something correct, this may be good enough for 9.1. And with the accessing statics through objects up in the air, it may never be needed. 

(and looking at how method_offset is used in #13053 I'm feeling even more confused, need to draw some diagrams and check out the code that builds the tables, if and when the time comes)

I can open a performance improvement issue about it if you like.